src/ws-dropdown/ws-dropdown.js

```diff
@@ -140,11 +140,11 @@ export class WSDropdown extends Component {
    * @returns {String}
    */
   getTextFromValue(value) {
-    let text = this.state.text;
+    let text = (this.state) ? this.state.text : '';
```

Comment:
this.state should never be empty, as it's assigned in the constructor. in which situation did this raise an error?